packages/plugin-survey-multi-choice/src/index.ts

```diff
@@ -233,7 +235,7 @@ class SurveyMultiChoicePlugin implements JsPsychPlugin<Info> {
     // render
     display_element.innerHTML = html;
 
-    document.querySelector("form").addEventListener("submit", (event) => {
+    document.querySelector(`form#${trial_form_id}`).addEventListener("submit", (event) => {
```

Comment:
i think it would just be more simple to instead directly use `querySelector` on `display_element`, which represents the container of the current experiment. just saves the hassle of using interpolation so much